wetmap/src/supabaseCalls/photoSupabaseCalls.js

```diff
@@ -117,7 +116,6 @@ export const getAnimalMultiSelect = async (text) => {
 //       .from("photos")
 //       .select()
 //       // .ilike("userName", "%" + value.myCreatures + "%")
-//       // .eq("month", value.sliderVal)
 //       .ilike("label", "%" + creatureListFinal + "%")
```

Comment:
Will some of these commented lines have use later? Or should we just delete them all

Reply:
looks like an old, now unused query, ill remove it